website_event_baidu_map/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Website Event Baidu Map
+=======================
+
+This module extends the functionality of website to support baidu map in the website event page.
+
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (website_event)
+
+
+Configuration
+
+=============
+
+To configure this module, you need to:
+
+* Go to Settings/Parameters/System Parameters and create a record which Key must be named baidu.api and the Value like [http://api.map.baidu.com/api]?v=[2.0]&ak=[your ak].
```

Comment:
Please add the description for how to get the application key for baidu map and test the module in https environment.